src/NuGet.Core/NuGet.CommandLine.XPlat/NuGet.CommandLine.XPlat.csproj

```diff
@@ -25,9 +25,9 @@
 
 <!-- Microsoft.Build.Locator is only used when debugging, and the compiler will skip copying this dependency from Release assemblies we insert because we only refer to it conditionally with the DEBUG configuration.
   Uncomment the following when debugging. Also uncomment the MSBuildLocator code from Program.cs -->
-  <!-- <ItemGroup>
+  <ItemGroup>
```

Comment:
I imagine you meant to revert this?

I really hate that we removed the Choose/When Debug, imo it's just causing problems for people.

Reply:
Created follow up issue: https://github.com/NuGet/Client.Engineering/issues/1941